pkg/geo/geos/geos.cc

```diff
@@ -1359,3 +1369,51 @@ CR_GEOS_Status CR_GEOS_Node(CR_GEOS* lib, CR_GEOS_Slice a, CR_GEOS_String* nodeE
   lib->GEOS_finish_r(handle);
   return toGEOSString(error.data(), error.length());
 }
+
+// Voronoi Diagram
```

Comment:
you can remove comments from these two  lines -- if you want them, add them in geos.h and explain in more detail about it.